broker-util/oo-admin-move

```diff
@@ -156,6 +156,8 @@ gear_uuid.split(',').each do |uuid|
         puts ne.message
       end
       rc |= 1
+    rescue OpenShift::QuotaExceededException => qe
+      puts qe.message
```

Comment:
This will break our json output mode.  Look at how we are handling OpenShift::UserException in the subsequent few lines.